test/Makefile

```diff
@@ -3,6 +3,14 @@ ifdef OPENMP
   CFLAGS += -fopenmp
 endif
 
+TARGET := $(shell $(CC) -dumpmachine)
```

Comment:
O_o

Ok, so I guess we're stuck with `-lrt`. But what about using `uname -s` as a slightly more portable platform check?